src/dpr/routes/journeys/view-report/utils.ts

```diff
@@ -133,13 +141,15 @@ const createQueryParamsFromFormData = ({
             case FilterType.dateRange.toLocaleLowerCase():
             case FilterType.granularDateRange.toLocaleLowerCase():
               {
-                let dateValue = <string>value
+                let dateValue: string | undefined = <string>value
```

Comment:
This cant be undefined at this point - you already checked for this on line 132 with `if (value && ...`